tgui/packages/tgui-panel/styles/goon/chat-dark.scss

```
@@ -900,6 +900,13 @@ em {
   margin-left: 3em;
 }
 
+.examine_block {
+  background: #1b1c1e;
+  border: 1px solid #a4bad6;
+  margin: 7px;
+  padding: 8px 12px;
+}
+
```

You should use `em` units to write these styles, one `em` is equal to whatever font size in px you have set in chat settings, but try to stick to "one unit of margin is 0.5em" convention that we have in tgui.

Suggestion:

```suggestion
.examine_block {
  background: #1b1c1e;
  border: 1px solid #a4bad6;
  margin: 0.5em;
  padding: 0.5em 0.75em;
}
```